.yarnrc

```diff
@@ -0,0 +1 @@
+yarn-path ".yarn/releases/yarn-1.16.0.js"
```

Comment:
I saw this in the Jest repo.
This should ensure that yarn always uses the given release version (which is included in the repo), so that no matter which images are installed on CI, yarn should consistently run with the version we want.

Reply:
How to those release versions end up in the `.yarn`-path? Will yarn pull them down in case they're missing? Are they cached?

Reply:
We manually download and save the file from the yarn release page (https://github.com/yarnpkg/yarn/releases/tag/v1.16.0)

Reply:
Ok, I just checked in CI that this works.

I can document how to deal with those

Reply:
See https://yarnpkg.com/en/docs/yarnrc#yarn-path-

> Instructs yarn to defer to another Yarn binary for execution. Useful if you want to bundle Yarn into your repository and have everyone use the same version for consistency. This was introduced in Yarn 1.0, so all developers must have Yarn >= 1.0 installed.